### PR TITLE
Fix bazel build for example cpp

### PR DESCRIPTION
**Pull request by netroby**
Signed-off-by: ZhiFeng Hu <hufeng1987@gmail.com>

### CODE REVIEW

File: examples/cpp/hello-lib.cc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include "examples/cpp/hello-lib.h"
+#include "hello-lib.h"
```
Comment (davidzchen) [inline examples/cpp/hello-lib.cc]: Why this change? The include path is relative to the root of the workspace.

File: examples/cpp/README.md [hunk 2]
```diff
@@ -2,3 +2,9 @@ C++ Examples
 ============
 
 These examples demonstrate building C++ binaries, libraries, and tests.
+
+To starting build , change to this directory.
+```
+cd examples/cpp/
+bazel build //:hello-world
```
Comment (davidzchen) [inline examples/cpp/README.md]: Have you tested this? The `:hello-world` target is in the `//examples/cpp` package. There is no `:hello-world` target in the root BUILD file.

**Comment by netroby** [comment]
Yes, i tested. to run bazel build, must cd examples/cpp/
and there be WORKSPACE file locate at examples/cpp/WORKSPACE

**Comment by kchodorow** [comment]
We don't want a WORKSPACE file in examples/cpp and the example works as-is, so this isn't a change we want to merge.

**Comment by netroby** [comment]
Ok. I just want to show a real example.
Do not need to clone the full bazel source code repository. 
Anyway, thanks for let me know
